src/_pytest/pytester.py

```diff
@@ -1695,8 +1691,8 @@ def collect_by_name(
     def popen(
         self,
         cmdargs,
-        stdout: Union[int, TextIO] = subprocess.PIPE,
-        stderr: Union[int, TextIO] = subprocess.PIPE,
+        stdout=subprocess.PIPE,
```

Comment:
Was the above incorrect?

Reply:
It depends on what it means to be incorrect. I removed the annotation here because this is the only method in `Testdir` that has some parameters annotated. And it makes sense because the docstring pretty much just says "please read the Pytester counterpart," and by removing the annotations makes the generated document easier to read. Would you agree? I can revert this part of the change if it is intended so.

Reply:
Generally if the type annotation is correct, I'd keep it in place. But I'm OK with either way.